components/channel_header/channel_header.js

```diff
@@ -78,6 +78,7 @@ class ChannelHeader extends React.PureComponent {
             openModal: PropTypes.func.isRequired,
             closeModal: PropTypes.func.isRequired,
         }).isRequired,
+        pinnedPostsCounts: PropTypes.number,
```

Comment:
minor nitpick: can we move this over the actions so the actions are the last thing on it? seems like a common pattern we follow.
also:
```suggestion
        pinnedPostsCount: PropTypes.number,
```